sdks/python/apache_beam/transforms/environments.py

```diff
@@ -37,6 +37,11 @@
            'SubprocessSDKEnvironment', 'RunnerAPIEnvironmentHolder']
 
 
+def _looks_like_json(config_string):
+  import re
+  return re.match(r'\s*\{.*\}\s*$', config_string)
```

Comment:
I know you're not responsible for this function, but I think it should be

```python
return re.match(r'\s*\{.*\}\s*$', config_string) is not None
```

Reply:
Ah, good call. Updated.